### ISSUE TITLE
Add nickname system for players

### ISSUE THREAD

**Issue by momo5502** [momo5502/w3m#125]
Nickname system for players, this will prove useful from RP perspective, as a person that didn't reveal their name will be simply shown as "Stranger", with the ability to set their name from the client's side to what we want

**Comment by momo5502** [comment]
This is partly supported :D

**Comment by werasik2aa** [comment]
Really ;D

**Comment by momo5502** [comment]
> Really? ;D

yes, it uses the steam username at the moment and displays it to other players as the overhead name

**Comment by werasik2aa** [comment]
oh i'm too tried dev the mod but was many time ago you can take something if you need.

https://github.com/werasik2aa/Witcher3-Multiplayer/tree/main
